pt/lessons/specifics/mnesia.md

```diff
@@ -0,0 +1,185 @@
+  - Devo armazenar dados em vários nós, em vez de um?
+  - Preciso escolher onde armazenar informações (RAM ou disco)?
+
+## Schema
+
+Como Mnesia faz parte do núcleo do Erlang, ao invés de Elixir, temos que acessá-lo com a sintaxe de dois pontos (Ver lição: [Erlang Interoperability](https://elixirschool.com/lessons/advanced/erlang/)) como a seguir:
+
+```shell
+
+iex> :mnesia.create_schema([node()])
+
+# or if you prefer the Elixir feel...
+
+iex> alias :mnesia, as: Mnesia
+iex> Mnesia.create_schema([node()])
+
+```
+
+Para esta lição, vamos tomar a última abordagem quando se trabalha com a API Mnesia. `Mnesia.create_schema/1` inicializa um novo schema vazio e passa em uma lista de nós. Neste caso, estamos passando o nó associado com a nossa sessão IEx.
+
+## <a name="nos"></a> Nós
+
+Uma vez que executar o comando`Mnesia.create_schema([node()])` via IEx, você deve ver uma pasta chamada **Mnesia.nonode@nohost** ou similar no seu diretório de trabalho atual. Você pode estar se perguntando o que o **nonode@nohost** significa já que não se deparamos com isso antes. Vamos dar uma olhada.
+
+```shell
+$ iex --help
+Usage: iex [options] [.exs file] [data]
+
+  -v                Prints version
+  -e "command"      Evaluates the given command (*)
+  -r "file"         Requires the given files/patterns (*)
+  -S "script"       Finds and executes the given script
+  -pr "file"        Requires the given files/patterns in parallel (*)
+  -pa "path"        Prepends the given path to Erlang code path (*)
+  -pz "path"        Appends the given path to Erlang code path (*)
+  --app "app"       Start the given app and its dependencies (*)
+  --erl "switches"  Switches to be passed down to Erlang (*)
+  --name "name"     Makes and assigns a name to the distributed node
+  --sname "name"    Makes and assigns a short name to the distributed node
+  --cookie "cookie" Sets a cookie for this distributed node
+  --hidden          Makes a hidden node
+  --werl            Uses Erlang's Windows shell GUI (Windows only)
+  --detached        Starts the Erlang VM detached from console
+  --remsh "name"    Connects to a node using a remote shell
+  --dot-iex "path"  Overrides default .iex.exs file and uses path instead;
+                    path can be empty, then no file will be loaded
+
+** Options marked with (*) can be given more than once
+** Options given after the .exs file or -- are passed down to the executed code
+** Options can be passed to the VM using ELIXIR_ERL_OPTIONS or --erl
+```
+
+Quando passamos a opção `--help` para IEx a partir da linha de comando, nos é apresentado todas as opções possíveis. Podemos ver que existe opção `--name` e `--sname` para atribuição de informações para nós. Um nó é apenas uma Máquina Virtual do Erlang lidando com suas próprias comunicações, bargabe collection, processamento agendado, memória e muito mais. O nó está sendo nomeado como **nonode@nohost** simplesmente por padrão.
```

Comment:
Typo on `bargabe collection`, which should be `garbage collection`. May this is a typo from the original text, but I didn't check.